lib/client-assets.php

```diff
@@ -246,6 +246,7 @@ function gutenberg_register_scripts_and_styles() {
 		'wp-data',
 		gutenberg_url( 'build/data/index.js' ),
 		array(
+			'utils',
```

Comment:
Maybe we can use `get_current_user_id` in the inline script to avoid loading another script just for a variable?

Reply:
@youknowriad the `utils` script is a dependency for other scripts (`/wp-includes/js/media-views.js`, `/wp-admin/js/common.js`, `/wp-admin/js/editor.js`) and will load anyway, it's just a matter of timing. 

However, if the intention is to move away from older libraries, then using just `get_current_user_id()` in the inline script makes sense.

Reply:
Gotcha, I think I have a small preference towards avoiding the dependency. Maybe at some point, these dependencies won't be needed anymore.

Reply:
Sounds good to me. I pushed up the change.